collections/_components/checkbox.html

```diff
@@ -1,7 +1,167 @@
 ---
 title: Checkbox
-description: Checkboxes allow the user to select one or more items from a list.
+description: This component is used when you require users to select multiple items from a list.
 layout: layout-sidenav
-category: form
-react: true
----
\ No newline at end of file
+category: general
+react-link: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/?path=/docs/components-form--basic
+vue-link: true
+github-link: true
+toc: true
+---
+
+{% capture checkbox-examples %}
+<div class="row">
+    <div class="col">
+        Default
+        <br />
+        <div class="form-check">
+            <input class="form-check-input" type="checkbox" value="" id="checkboxDefault" />
+            <label class="form-check-label" for="checkboxDefault">
+                checkbox
```

Comment:
Remove
```
Default 
<br\>
```
and replace the title using the checkbox label, same for the rest of the checkboxes
```
 <label class="form-check-label" for="checkboxDefault">
                Default checkbox
 </label>

Reply:
and also the radio buttons pages :)